src/java.base/share/classes/java/security/doc-files/security-related-system-properties.html

```diff
@@ -0,0 +1,192 @@
+ questions.
+-->
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>The java.security.debug system property</title>
+    <style>
+        #debug {
+         border: 1px solid black;
+         border-collapse: collapse;
+         margin: 0 auto;
+        }
+        #debug caption {
+          font-weight: bold;
+          font-size: smaller;
+        }
+        #debug, #debug th, #debug td {
+         border: 1px solid black;
+         padding: 2px .5em;
+        }
+        #debug tbody th {
+         font-weight: normal;
+         text-align:left;
+        }
+    </style>
+</head>
+<body LANG="en-US" DIR="LTR">
+<H1 style="text-align:center">The java.security.debug system property</H1>
+<a id="security.debug"></a>
+<H2>Debug</H2>
+<p><B>{@systemProperty java.security.debug}</B><BR></p>
+<p> To monitor security access, you can set the <code>java.security.debug</code> system property,
+    which determines what trace messages are printed during execution. The value of the property is one or more options separated by a comma.
+    Some options also have additional sub-options. Sub-options are specified by appending a ":" to the option, followed by a list of one more sub-options separated by a comma.
+    For example, to specify the <code>ocsp</code> and <code>verbose</code> options with the <code>certpath</code> option: set the property to "<code>certpath:ocsp,verbose</code>".
+</p>
+
+<H3>Printing Thread and Timestamp Information</H3>
+<p>
+    You can append the following strings to the value specified in the <code>java.security.debug</code>
+    system property to print additional information:
+<ul>
+    <li><code>+thread</code>:Print thread and caller information</li>
+    <li><code>+timestamp</code>:Print timestamp information</li>
+</ul>
+<p>
+    For example, to add thread, caller, and timestamp information to all debuging output,
+    set the <code>java.security.debug</code> system property on the command line as follows:
+    <pre><code>java -Djava.security.debug=all+thread+timestamp MyApp</code></pre>
+
+<p>The following table lists <code>java.security.debug</code> options table:</p>
+
+<table id="debug">
+    <caption><b>Security Debug Options</b></caption>
+    <thead>
+    <tr>
+        <th scope="col">Option</th>
+        <th scope="col">Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <th scope="row"><code>all</code></th>
+        <td>Turn on all the debugging options</td>
+    </tr>
+    <tr>
+        <th scope="row"><code>certpath</code></th>
+        <td>Turns on debugging for the PKIX <code>CertPathValidator</code> and <code>CertPathBuilder</code> implementations. Use the <code>ocsp</code> sub-option with the <code>certpath</code> option for OCSP protocol tracing. A hexadecimal dump of the OCSP request and response bytes is displayed.
+            You can use the following sub-options with the <code>certpath</code> option:
```

Comment:
Change this to: "The following sub-options can be used with the <code>certpath</code> option:"